app/src/main/java/com/vero/tasky/agenda/data/workmanagerrunner/GetFullAgendaWorkManagerRunnerImpl.kt

```diff
@@ -0,0 +1,35 @@
+package com.vero.tasky.agenda.data.workmanagerrunner
+
+import androidx.work.Constraints
+import androidx.work.ExistingWorkPolicy
+import androidx.work.NetworkType
+import androidx.work.OneTimeWorkRequestBuilder
+import androidx.work.WorkManager
+import com.vero.tasky.agenda.domain.worker.GetFullAgendaWorker
+import com.vero.tasky.agenda.domain.workmanagerrunner.GetFullAgendaWorkManagerRunner
+import javax.inject.Inject
+
+class GetFullAgendaWorkManagerRunnerImpl @Inject constructor(
```

Comment:
"WorkManagerRunner" can be replaced with "Worker"
It's the same, but with less wods